webpack.config.production.js

```diff
@@ -15,26 +15,15 @@ const config = {
         publicPath: '../dist/'
     },
 
-    module: {
-        ...baseConfig.module,
-
-        loaders: [
-            ...baseConfig.module.rules,
-        ]
-    },
-
     plugins: [
         ...baseConfig.plugins,
         new webpack.DefinePlugin({
             __DEV__: false,
             'process.env.NODE_ENV': JSON.stringify('production')
         }),
-        new webpack.optimize.UglifyJsPlugin({
-            compressor: {
-                screw_ie8: true,
-                warnings: false
-            }
-        }),
+        new webpack.LoaderOptionsPlugin({
+            minimize: true
```

Comment:
huh, interesting. which plugins still have a `minimize` attribute?

Reply:
I've just removed them from production and web.